classes/Mission.php

```diff
@@ -320,4 +363,47 @@ public static function maxMissionRank(int $player_rank): int {
         }
         return $max_mission_rank;
     }
-}
\ No newline at end of file
+
+    public static function processRewards(Mission $mission, User $player, System $system): string {
+        if (count($mission->rewards) > 0) {
+            try {
+                $reward_text = "";
+                // load player inventory
+                $player->getInventory();
+
+                // get items from DB
+                $reward_keys = [];
+                foreach ($mission->rewards as $item) {
+                    $reward_keys[] = $item['item_id'];
+                }
+                $reward_key_string = '(' . implode(',', $reward_keys) . ')';
+                /** @var Item[] $reward_items */
+                $reward_items = array();
+                $result = $system->db->query("SELECT * FROM `items` where `item_id` IN {$reward_key_string}");
+                while ($row = $system->db->fetch($result)) {
+                    $reward_items[$row['item_id']] = Item::fromDb($row);
+                }
+                // roll RNG, add to inventory
+                foreach ($mission->rewards as $item) {
+                    if (rand(0, 100) <= $item['chance']) {
```

Comment:
```suggestion
                    if (mt_rand(0, 100) <= $item['chance']) {
```
Supposedly mt_rand is better than rand somehow. ALthough we could also use [random_int](https://www.php.net/manual/en/function.random-int.php)